opensquirrel/ir.py

```diff
@@ -656,14 +681,18 @@ def accept(self, visitor: IRVisitor) -> None:
             statement.accept(visitor)
 
 
+# Type Aliases
+AxisLike = Union[ArrayLike, Axis]
+QubitLike = Union[SupportsInt, Qubit]
+
+
 ANNOTATIONS = {
```

Comment:
I know this is not part of this PR. But any idea why `ANNOTATIONS` is spelled all in capitals? Is it because it's a constant dictionary? Is this a proper name also? Maybe `typename_to_type_map` as a suggestion? Or  `TYPENAME_TO_TYPE_MAP`?

Reply:
But, as I commented elsewhere, I would consider turning it into a class with class methods, like `is_qubit_like(str) -> bool`.